.circleci/config.yml

```diff
@@ -1385,3 +1407,30 @@ workflows:
       equal: [bump, << pipeline.parameters.action >>]
     jobs:
       - release-train
+
+  manual-test-workflow:
+    when:
+      equal: ["run-manual-tests", << pipeline.parameters.action >>]
```

Comment:
to trigger, just go into circleCI, select a branch and click trigger pipeline, then enter the value: 
<img width="797" alt="image" src="https://github.com/RevenueCat/purchases-ios/assets/3922667/14f9c32d-6a77-493b-b2c5-e86d8f11790b">

Reply:
we can get fancier by adding a github actions hook from a comment or tag that triggers this programmatically. Not sure it's worth it though since this isn't exactly hard to do. Thoughts?

Reply:
FWIW I did test triggering this manually and it works

Reply:
it'd also be quite easy to go beyond this and separate out backend integration tests, tests for iOS versions, etc in case we ever wanted to specifically run those, but it feels like this gets us 90% of the way there with 10% of the work

Reply:
what do we currently do for `bump` on tags? This mimics the way that the bump job was set up

Reply:
why would you need to run these tests on a tag, though? wouldn't we usually run tests _before_ tagging?